panel/reactive.py

```diff
@@ -555,10 +559,13 @@ def jscallback(self, args={}, **callbacks):
             callbacks[k] = self._rename.get(v, v)
         return Callback(self, code=callbacks, args=args)
 
-    def jslink(self, target, code=None, args=None, bidirectional=False, **links):
+    def jslink(self, target, code: Dict[str,str]=None,
```

Comment:
Here I don't now the relevant Union of target types.

Reply:
Difficult one, I think it's `Union[bokeh.model.Model, holoviews.core.Dimensioned, panel.viewable.Renderable, panel.viewable.Viewer]`

Reply:
Actually I think you can also link a simple `param.Parameterized`, will have to double check.